lib/generated_definitions/au.rb

```diff
@@ -27,9 +27,9 @@ def self.holidays_by_month
             {:wday => 1, :week => 2, :name => "Labour Day", :regions => [:au_vic]},
             {:function => "march_pub_hol_sa(year)", :function_arguments => [:year], :name => "March Public Holiday", :regions => [:au_sa]},
             {:wday => 1, :week => 2, :name => "Canberra Day", :regions => [:au_act]}],
-      4 => [{:mday => 25, :name => "ANZAC Day", :regions => [:au]},
-            {:mday => 25, :observed => "to_monday_if_sunday(date)", :observed_arguments => [:date], :name => "ANZAC Day", :regions => [:au_nsw, :au_vic, :au_qld, :au_nt, :au_sa, :au_tas]},
-            {:mday => 25, :observed => "to_monday_if_weekend(date)", :observed_arguments => [:date], :name => "ANZAC Day", :regions => [:au_wa, :au_act]}],
+      4 => [{:mday => 25, :name => "ANZAC Day", :regions => [:au_nsw, :au_sa, :au_tas, :au_vic, :au_wa]},
```

Comment:
Did this need to be changed as well btw?

Reply:
Yeah looked at 2020, WA Anzac day 
![image](https://user-images.githubusercontent.com/55837459/100853167-2e08bb00-34d3-11eb-9e7d-a485bb1dd7ae.png)

And 2021 throughout some states have an additional for ANZAC as well
![Screen Shot 2020-12-02 at 7 19 18 pm](https://user-images.githubusercontent.com/55837459/100853277-4f69a700-34d3-11eb-9277-3b76cfb16765.png)
![Screen Shot 2020-12-02 at 7 18 22 pm](https://user-images.githubusercontent.com/55837459/100853271-4e387a00-34d3-11eb-9831-b91a624be20b.png)